core/src/main/java/io/dekorate/utils/Ports.java

```diff
@@ -75,15 +82,15 @@
   }
 
   public static Port populateHostPort(Port port) {
-    if (!isWebPort(port)) {
+    if (!isWebPort(port) && !isNodePort(port)) {
```

Comment:
It's not obvious to me why were are also checking `isNodePort`. A comment might help.

Reply:
I removed this check in the last commit, I think that a node port service is similar to a cluster IP one, a connection received on the port will be forwarded to a pod so we need to populate the host port also in that case. This is also the reason why the `PopoluateWebPort` configurator [is added](https://github.com/aureamunoz/dekorate/blob/0831626fb58707f0eca9f1d9bcdc7e839f04dcbe/annotations/minikube-annotations/src/main/java/io/dekorate/minikube/config/DefaultMinikubeConfigGenerator.java#L32) in the Minikube module